cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/mapping/ResourceProvider.java

```diff
@@ -0,0 +1,20 @@
+package cloud.foundry.cli.crosscutting.mapping;
+
+import java.io.InputStream;
+
+/**
+ * Class used for accessing information from property files.
+ */
+public class ResourceProvider {
```

Comment:
I am pretty sure we have this kind of code in use in other classes. While you're at it, please consider switching those over to your new class, too.

Reply:
Right now only in the CfArgumentsCreator. Not sure how we continue with this code there anyways..

Reply:
Good point. We should probably wait for #154 to be merged.

Reply:
You mean #150. 
The usage was with that PR removed, so there is no other usage